packages/events/test/EventSystem.tests.ts

```diff
@@ -1,4 +1,4 @@
-import { Renderer } from '@pixi/core';
```

Comment:
The events package `README.md` still has a reference to `delete Renderer.__plugins.interaction;`. Should this be updated to `extensions.remove(InteractionManager)`?